### PR TITLE
CI: compile using wake 0.18.1

### PR DESCRIPTION
**Pull request by terpstra**
**Type of change**: other enhancement
**Impact**: no functional change
**Development Phase**:  implementation

Stop regressing on wake 0.18 support; compile using 0.18.1 instead of 0.17.2.

### CODE REVIEW

**Comment by terpstra** [comment]
This one turned out to be very easy.

File: ci-tests/wake_scala_compilation [hunk 1]
```diff
@@ -34,5 +34,5 @@ echo "Compile Scala"
 
 export WAKE_PATH=$PATH
 wake --init .
-wake --no-tty -p1 -dv 'compileScalaModule rocketchipScalaModule | getPathResult'
+wake -x 'compileScalaModule rocketchipScalaModule | getPathResult'
```
Comment (richardxia) [inline ci-tests/wake_scala_compilation:37]: Is there a reason you got rid of the `--no-tty`? I think this affects the carriage return thing for "progress bars", and in CI, when you don't really have a real TTY, it often ends up creating a newline for each carriage return as well.

Comment (terpstra) [reply ci-tests/wake_scala_compilation:37]: Wake does not need to be told there is no tty. It auto-detects this.
The only reason to use --no-tty is if you actually DO have a tty, but don't want the bards.

Comment (richardxia) [reply ci-tests/wake_scala_compilation:37]: Ah, I see that you're right. I took a quick look at the actual [CI console log from this PR](https://github.com/chipsalliance/rocket-chip/pull/2584/checks?check_run_id=946222730) and saw some progress bars, but after taking a closer look, I see that they're actually from `curl`.

**Pull request merged**